components/director/internal/features/features.go

```diff
@@ -2,6 +2,7 @@ package features
 
 // Config missing godoc
 type Config struct {
-	DefaultScenarioEnabled bool   `envconfig:"default=true,APP_DEFAULT_SCENARIO_ENABLED"`
-	ProtectedLabelPattern  string `envconfig:"default=.*_defaultEventing,APP_PROTECTED_LABEL_PATTERN"`
+	DefaultScenarioEnabled  bool     `envconfig:"default=true,APP_DEFAULT_SCENARIO_ENABLED"`
+	ProtectedLabelPattern   string   `envconfig:"default=.*_defaultEventing,APP_PROTECTED_LABEL_PATTERN"`
+	SecuredApplicationTypes []string `envconfig:"default=empty,APP_SECURED_APPLICATION_TYPES"`
```

Comment:
Is the envconfig library works with slices and how?

Is it comma-separated in the chart (currently we have only one value, but I am curious how it will work in the future)

Probably it is a good idea to document it with a comment in the chart or config repo in order to streamline the maintance in the future.

Reply:
Yes, multiple values are passed by setting a comma-separated string.